Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -119,22 +122,36 @@ public bool TryGetXComponentCorrelationId(string instrumentationKey, out string
                             {
                                 this.GenerateCorrelationIdAndAddToDictionary(instrumentationKey, appId.Result);
                             }
+#if !NETCORE
                             catch (Exception ex)
                             {
                                 CrossComponentCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
                             }
+#else
+                            catch (Exception)
```

Comment:
The CrossComponentCorrelationEventSource is not available for .NETStandard1.6. What other logging options are available?